execution/executor/src/components/chunk_output.rs

```diff
@@ -245,6 +245,178 @@ impl ChunkOutput {
 }
 
 pub fn update_counters_for_processed_chunk(
+    transactions: &[SignatureVerifiedTransaction],
```

Comment:
cannot there be some trait that will allow these two functions to be merged?

Reply:
Yes, I intended to do this as a follow up, when I move this counter off of critical path. But will add this in this PR.